src/android/app/src/main/res/menu/menu_in_game.xml

```diff
@@ -27,6 +27,11 @@
         android:icon="@drawable/ic_fit_screen"
         android:title="@string/emulation_switch_screen_layout" />
 
+    <item
+        android:id="@+id/menu_portrait_screen_layout"
+        android:icon="@drawable/ic_fit_screen"
```

Comment:
A unique icon should be used for the portrait layout options. A simple 90° rotation of the existing logo should suffice.